docs/sources/flow/reference/cli/run.md

```diff
@@ -75,3 +75,17 @@ other agent nodes; if this is empty it will attempt to find a suitable address
 to advertise from a list of default network interfaces. The agent must be
 reachable over HTTP on this address as communication happens over the agent's
 HTTP server.
+
+## Converter mode
```

Comment:
"Mode" tends to be reserved for operational modes (static mode, flow mode), so I don't want to overload the term here. 

Can we just call it what it is? Something like "Configuration conversion?"

Reply:
done